ArchUnitNET/Fluent/Syntax/Elements/Members/MethodMembers/MethodMembersShouldThat.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ArchUnitNET.Domain;
+using static ArchUnitNET.Fluent.Syntax.ActivatorHandler;
+
+namespace ArchUnitNET.Fluent.Syntax.Elements.Members.MethodMembers
+{
+    public class MethodMembersShouldThat<TRuleTypeShouldConjunction, TRuleType> :
```

Comment:
please, adapt class name

Reply:
renamed to ShouldRelatedToMethodMembersThat in new version